android/src/main/java/com/chirag/RNMail/RNMailModule.java

```diff
@@ -41,13 +42,13 @@ public String getName() {
    */
   private String[] readableArrayToStringArray(ReadableArray r) {
     int length = r.size();
-    String[] recipients = new String[length];
+    String[] arr = new String[length];
```

Comment:
ah. ok. check new commit; renamed method to clarify the intent (to convert a recipients readable arr to string array).